io.edgehog.devicemanager.HardwareInfo.json

```diff
@@ -0,0 +1,40 @@
+{
+  "interface_name": "io.edgehog.devicemanager.HardwareInfo",
+  "version_major": 1,
+  "version_minor": 0,
+  "type": "properties",
+  "ownership": "device",
+  "description": "General hardware capabilities",
+  "mappings": [
+    {
+      "endpoint": "/cpu/architecture",
+      "type": "string",
+      "description": "CPU Architecture "
+    },
+    {
+      "endpoint": "/cpu/model",
+      "type": "string",
+      "description": "CPU Model Code"
+    },
+    {
+      "endpoint": "/cpu/modelName",
+      "type": "string",
+      "description": "CPU Model Display Name"
+    },
+    {
+      "endpoint": "/cpu/vendor",
+      "type": "string",
+      "description": "CPU Vendor"
+    },
+    {
+      "endpoint": "/mem/total",
+      "type": "integer",
+      "description": "Total RAM quantity (MB)"
+    },
+    {
+      "endpoint": "/storage/total",
+      "type": "integer",
+      "description": "Total storage quantity (MB)"
+    }
```

Comment:
let's remove this, it gets tricky when there is more than one store unit, so I would postpone it to a dedicated interface.